pages/user/[username]/card.tsx

```diff
@@ -1,5 +1,11 @@
-import DevCard from "components/molecules/DevCard/dev-card";
+import Button from "components/atoms/Button/button";
+import HeaderLogo from "components/molecules/HeaderLogo/header-logo";
+import DevCardCarousel from "components/organisms/DevCardCarousel/dev-card-carousel";
+import BubbleBG from "../../../img/bubble-bg.svg";
+import { STUB_DEV_CARDS } from "components/organisms/DevCardCarousel/stubData";
 import { GetServerSideProps, NextPage } from "next";
+import { useEffect, useState } from "react";
+import { useTransition, animated, useSprings, easings } from "react-spring";
```

Comment:
nit: keep third-party imports at the top (not a blocker)

```suggestion
import { GetServerSideProps, NextPage } from "next";
import { useEffect, useState } from "react";
import { useTransition, animated, useSprings, easings } from "react-spring";
```